### DISCUSSION TITLE
Can't set address-delimiter

### DISCUSSION THREAD

**Discussion by debtvision-link** [arkmq-org/arkmq-org-broker-operator#959] [General] [answered]
**Describe the bug**
In the [official documentation](https://activemq.apache.org/components/artemis/documentation/latest/wildcard-syntax.html#wildcard-syntax), it is described that you can customize the wildcard delimiter by adding the following config to broker.xml:
```xml
<wildcard-addresses>
   <delimiter>.</delimiter>
</wildcard-addresses>
```

What would this config look like when using artemiscloud? I tried setting it via the custom broker properties, but it didn't work:

```yaml
# ...
  brokerProperties:
    - "wildcardAddresses.delimiter=/"
# ...
```

I also neither couldn't find any properties related to the address-delimiter, nor how to add custom broker.xml config.

> [!TIP]
> Vote this issue reacting with :+1: or :-1:

**Comment by brusdev** [comment, marked as answer]
@debtvision-link this is a question for the Apache ActiveMQ Artemis [users mailing list](https://activemq.apache.org/contact) because the [broker properties](https://activemq.apache.org/components/artemis/documentation/latest/configuration-index.html#broker-properties) are a feature of  Apache ActiveMQ Artemis.

In your case, the following example should work:
```
  brokerProperties:
    - 'wildcardConfiguration.delimiter=/'
```
